graph-db/README.md

```diff
@@ -37,6 +37,17 @@ docker run --rm -it -p 8182:8182 tinkerpop/gremlin-server
 
 If you want to load existing local data into Gremlin Server, you can add `-v <path_to_cmr_graph-db>/data:/data` to map a local directory with data to the docker instance. Replace the `<path_to_cmr_graphdb>` with the path to your local data directory.
 
+The Gremlin server can be started with multiple configurations by default it uses a websockets but, it can be configured to use http by running:
```

Comment:
is this a run on sentence?

Reply:
Yea reading it again, I think it should get broken up after 'configurations'